Manifest/Manifest.1.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version='1.0' encoding='utf-8'?>
+<ExtensionPackages Version='1.0.0.0' Compression='none'>
```

Comment:
You mean in the tests? Or generally?
Generally it is easier to just use `!lua` instead of writing the full path of course, and in tests it is kind of a way to test the manifest.

Reply:
I meant the tests.
OK.